bodhi/server/services/updates.py

```diff
@@ -536,6 +538,24 @@ def new_update(request):
 
             if len(releases) > 1:
                 result = dict(updates=updates)
+
+            if from_tag:
+                koji = buildsys.get_session()
+                # Validate that <koji_tag>-pending-signing and <koji-tag>-testing exists,
+                # if not create them
+                koji_tag_pending_signing = from_tag + config.config.get(
+                    'koji_pending_signing_side_tag')
+                koji_tag_testing = from_tag + config.config.get('koji_testing_side_tag')
+                if not koji.getTag(koji_tag_pending_signing):
+                    koji.createTag(koji_tag_pending_signing, parent=from_tag)
+                if not koji.getTag(koji_tag_testing):
+                    koji.createTag(koji_tag_testing, parent=from_tag)
+
+                to_tag = koji_tag_pending_signing
+                # Move every new build to <from_tag>-pending-signing tag
+                for build in builds:
```

Comment:
I think you could use the update `add_tag` method here (https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L2845).

Reply:
I didn't noticed that update has `add_tag` method.